Classes/Ship.py

```diff
@@ -0,0 +1,61 @@
+from Classes.Square import Square
+
+
+class Ship:
+
+    def __init__(self, _id, size):
+        self.id = _id
+        self.size = size
+        self.m_factor = self.size - 1
+        # m is a factor that is used to calculate the ship's needed space so it can be placed on the board and is used
+        # by the place_ship functions later on
+        self.square_start = None
+        self.square_end = None
+        self.start_x = None
+        self.end_x = None
+        self.start_y = None
+        self.end_y = None
+
+    """Defines the position of the specific ship and adds it to the relevant dictionary accordingly"""
```

Comment:
Here you have the msg before the function and later on inside the function `is_ship_sunk`